docs/DeveloperGuide.md

```diff
@@ -1158,45 +1070,17 @@ testers are expected to do more *exploratory* testing.
 
 ### Adding a contact
 
-1. Adding a contact while all contacts are being shown
-
-   1. Prerequisites: List all contacts using the `view_contacts` command. Multiple contacts in the list.
+1. A contact can be added from any page.
 
    1. Test case: `add_contact n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/vendors t/clients`<br>
       Expected: Contact named John Doe with the respective details created, provided that there is no contact named John Doe or has the phone number 98765432. Returns back to home page.
 
-   1. Test case: `add_contact n/John  Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/vendors t/clients`<br>
+   1. Test case: `add_contact n/John &nbsp;&nbsp; Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 t/vendors t/clients`<br>
```

Comment:
It is printed as `&nbsp;`.  Maybe we can find a different way to show the double whitespace.